docs/specs/nsid.md

```diff
@@ -0,0 +1,71 @@
+# NameSpaced IDs (NSIDs)
```

Comment:
thoughts on adding a note to this about what we discussed this morning re: no implied hierarchy with sub domains?

Reply:
Done